lms/static/sass/elements/_icons.scss

```diff
@@ -0,0 +1,11 @@
+.verified-cert-icon {
+  @include float(left);
+  @include margin-right($baseline*0.4);
+
+  margin-top: ($baseline/4);
+  background: url('#{$static-path}/images/icon-sm-verified.png') no-repeat;
```

Comment:
I dug a little and found it in the LMS images directory. I noticed you pinged Roderick about an updated image. We can add it in a separate PR if necessary.